src/work-allocation/containers/task-manager-list/task-manager-list.component.spec.ts

```diff
@@ -90,7 +90,12 @@ describe('TaskManagerListComponent', () => {
     expect(headerCells).toBeDefined();
     expect(headerCells.length).toEqual(fields.length + 1); // Extra one for Manage +;
     for (let i = 0; i < fields.length; i++) {
-      expect(headerCells[i].textContent).toEqual(fields[i].columnLabel);
+      // ensure derivedIcon has no header and every other field does
+      if (fields[i].name !== 'derivedIcon') {
```

Comment:
I think a better condition would be `if (fields[i].columnLabel)` as it's more generic.

Reply:
Agreed so have changed